src/main/scala/cognite/spark/SdkV1Rdd.scala

```diff
@@ -20,7 +20,7 @@ case class SdkV1Rdd[A](
     @transient override val sparkContext: SparkContext,
     config: RelationConfig,
     toRow: A => Row,
-    uniqueId: A => Long,
+    uniqueId: A => String,
```

Comment:
Could we generalize to `SdkV1Rdd[A, I]` and have this be `uniqueId: A => I`, `processedItems: Set[I]` etc.?

Reply:
For sure! Wanted to avoid another Generic for `SdkV1Relation` and `SdkV1Rdd`, but seems like it's better than the alternative.